antarest/core/tasks/service.py

```diff
@@ -97,12 +104,27 @@ def add_task(
                 owner_id=request_params.user.impersonator,
             )
         )
-        future = self.threadpool.submit(self._run_task, action, task)
+
+        if event_messages is not None:
+            self.event_bus.push(
+                Event(
+                    EventType.TASK_STARTED,
+                    TaskEventPayload(
+                        id=task.id, message=event_messages.start
+                    ).dict(),
+                )
+            )
```

Comment:
the event should be sent even without messages, with a default message "Task {task.id} added".
Same for other events (running, failed, completed).
In that case, the "event_messages" should be renamed "custom_event_messages" (same for the object model) to reflect the fact that this is only customization, but the event are sent with default messages.